CMakeLists.txt

```diff
@@ -60,7 +60,8 @@ if(CMAKE_Fortran_COMPILER_ID MATCHES "GNU" AND CMAKE_BUILD_TYPE MATCHES "Debug")
     -ffpe-trap=invalid,zero,overflow
     -fimplicit-none
     -fbacktrace
-    -fmax-errors=0)
+    -fmax-errors=0
+    -Wno-maybe-uninitialized)
```

Comment:
I don't think we necessarily want to add this flag. The special pointer case that the compiler doesn't detect is annoying, but we could miss other things by adding this flag.

Reply:
I'm fine removing it. It's just annoying. It appears to be a known bug in gfortran/gcc.